code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -540,21 +565,68 @@
 
 	if(!istype(patient))
 		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
-		to_chat(user, span_notice("\The [src] makes a sad buzz and briefly displays a frowny face, indicating it can't scan [patient]."))
+		to_chat(user, span_notice("\The [src] makes a sad buzz and briefly displays an unhappy face, indicating it can't scan [patient]."))
+		emotion = AID_EMOTION_SAD
+		update_appearance(UPDATE_OVERLAYS)
 		return
 
 	woundscan(user, patient, src)
+	update_appearance(UPDATE_OVERLAYS) // woundscan updates emotion
+	flick(icon_state + "_pinprick", src)
+
+/obj/item/healthanalyzer/wound/update_overlays()
+	. = ..()
+	switch(emotion)
+		if(AID_EMOTION_HAPPY)
+			. += mutable_appearance(icon, "+no_wounds")
+		if(AID_EMOTION_WARN)
+			. += mutable_appearance(icon, "+wound_warn")
+		if(AID_EMOTION_ANGRY)
+			. += mutable_appearance(icon, "+angry")
+		if(AID_EMOTION_SAD)
+			. += mutable_appearance(icon, "+fail_scan")
+	if(emotion != AID_EMOTION_NEUTRAL)
+		addtimer(CALLBACK(src, PROC_REF(reset_emotions)), 2 SECONDS)
```

Comment:
This probably shouldn't be in `update_overlays()`
This is called from many places and should have no side effects.

I would suggest instead making a wrapper proc, like so:
```
/// Sets a new emotion display on the scanner
/obj/item/healthanalyzer/wound/proc/show_emotion(new_emotion)
	emotion = new_emotion
	update_appearance(UPDATE_OVERLAYS)
	if (emotion != AID_EMOTION_NEUTRAL)
		addtimer(CALLBACK(src, PROC_REF(show_emotion), AID_EMOTION_NEUTRAL), 2 SECONDS)
```